mcs/class/System.Data/mobile_referencesource.sources

```diff
@@ -270,23 +270,23 @@ MobileStubs.cs
-../referencesource/System.Data/System/Data/SQLTypes/SQLBinary.cs
-../referencesource/System.Data/System/Data/SQLTypes/SQLBoolean.cs
-../referencesource/System.Data/System/Data/SQLTypes/SQLByte.cs
-../referencesource/System.Data/System/Data/SQLTypes/SQLBytes.cs
-../referencesource/System.Data/System/Data/SQLTypes/SQLChars.cs
-../referencesource/System.Data/System/Data/SQLTypes/SqlCharStream.cs
-../referencesource/System.Data/System/Data/SQLTypes/SQLDateTime.cs
-../referencesource/System.Data/System/Data/SQLTypes/SQLDecimal.cs
-../referencesource/System.Data/System/Data/SQLTypes/SQLDouble.cs
-../referencesource/System.Data/System/Data/SQLTypes/SQLGuid.cs
-../referencesource/System.Data/System/Data/SQLTypes/SQLInt16.cs
-../referencesource/System.Data/System/Data/SQLTypes/SQLInt32.cs
-../referencesource/System.Data/System/Data/SQLTypes/SQLInt64.cs
-../referencesource/System.Data/System/Data/SQLTypes/SQLMoney.cs
-../referencesource/System.Data/System/Data/SQLTypes/SQLResource.cs
-../referencesource/System.Data/System/Data/SQLTypes/SQLSingle.cs
-../referencesource/System.Data/System/Data/SQLTypes/SQLString.cs
-../referencesource/System.Data/System/Data/SQLTypes/SQLUtility.cs
-../referencesource/System.Data/System/Data/SQLTypes/SqlXml.cs
+../../../external/corefx/src/System.Data.Common/src/System/Data/SQLTypes/INullable.cs
+../../../external/corefx/src/System.Data.Common/src/System/Data/SQLTypes/SQLBinary.cs
+../../../external/corefx/src/System.Data.Common/src/System/Data/SQLTypes/SQLBoolean.cs
+../../../external/corefx/src/System.Data.Common/src/System/Data/SQLTypes/SQLByte.cs
+../../../external/corefx/src/System.Data.Common/src/System/Data/SQLTypes/SQLBytes.cs
+../../../external/corefx/src/System.Data.Common/src/System/Data/SQLTypes/SQLChars.cs
+../../../external/corefx/src/System.Data.Common/src/System/Data/SQLTypes/SqlCharStream.cs
+../../../external/corefx/src/System.Data.Common/src/System/Data/SQLTypes/SQLDateTime.cs
+../../../external/corefx/src/System.Data.Common/src/System/Data/SQLTypes/SQLDecimal.cs
+../../../external/corefx/src/System.Data.Common/src/System/Data/SQLTypes/SQLDouble.cs
+../../../external/corefx/src/System.Data.Common/src/System/Data/SQLTypes/SQLGuid.cs
+../../../external/corefx/src/System.Data.Common/src/System/Data/SQLTypes/SQLInt16.cs
+../../../external/corefx/src/System.Data.Common/src/System/Data/SQLTypes/SQLInt32.cs
+../../../external/corefx/src/System.Data.Common/src/System/Data/SQLTypes/SQLInt64.cs
+../../../external/corefx/src/System.Data.Common/src/System/Data/SQLTypes/SQLMoney.cs
+../../../external/corefx/src/System.Data.Common/src/System/Data/SQLTypes/SQLResource.cs
+../../../external/corefx/src/System.Data.Common/src/System/Data/SQLTypes/SQLSingle.cs
+../../../external/corefx/src/System.Data.Common/src/System/Data/SQLTypes/SQLString.cs
+../../../external/corefx/src/System.Data.Common/src/System/Data/SQLTypes/SQLUtility.cs
+../../../external/corefx/src/System.Data.Common/src/System/Data/SQLTypes/SqlXml.cs
```

Comment:
Please make sure .sources files end with newlines (it screws up some part of our build on Linux otherwise).